src/catkin_pkg/package.py

```diff
@@ -421,7 +414,14 @@ def parse_package_for_distutils(path=None):
 
 
 class InvalidPackage(Exception):
-    pass
+    def __init__(self, msg, package_path=None):
+        self.msg = msg
+        self.package_path = package_path
+        Exception.__init__(self, self.msg)
+
+    def __str__(self):
+        result = '' if not self.package_path else 'Error(s) in package \'%s\':\n' % self.package_path
```

Comment:
Please avoid escaping in the string. To using single quotes within the string use double quotes for the string itself:

```
"Error(s) in package '%s':\n"
```

Reply:
Fixed.